public/app/percona/check/CheckPanel.test.tsx

```diff
@@ -50,26 +45,25 @@ describe('CheckPanel::', () => {
     jest.resetAllMocks();
   });
 
-  it('should show a message to unauthorized users', async () => {
-    const UnauthorizedError = () => ({
-      response: {
-        status: 401,
-      },
-    });
+  it('should show tabs for all checks and for failed checks', async () => {
+    let wrapper: ReactWrapper = {} as ReactWrapper;
 
-    const spy = jest.spyOn(SettingsService, 'getSettings').mockImplementation(() => {
-      throw UnauthorizedError();
+    await asyncAct(() => {
```

Comment:
Shouldn't this be `getMount` instead?

Reply:
Yes @fabio-silva, thanks for noticing it, much cleaner now!